astropy/units/format/base.py

```diff
@@ -108,7 +97,7 @@ def _format_fraction(
 
     @classmethod
     def to_string(
-        cls, unit: UnitBase, *, fraction: bool | Literal["inline", "multiline"] = True
```

Comment:
Is this intended ?
- the argument is passed to `_to_string_helper` which still supports `"multiline"`
- the docstring still lists it as valid

Reply:
`_to_string_helper()` has to accept `fraction="multiline"` because `Console.to_string()` might call it that way, but `Base.to_string()` never does. Besides, `_to_string_helper()` only cares if `fraction` is `False` or not.

The `Base.to_string()` docstring does list `"multiline"` as a valid value for `fraction`, but it also does that on `main` and it's equally wrong there too. It is my intent to keep non-trivial refactoring (which invisible for end users) separate from public documentation changes.

Reply:
I see. Can you please open an issue to remind ourselves to fix that docstring later then ?